### DISCUSSION TITLE
Live Stats, Scroll Bars and Title Bar, Modern Theme for Microsoft Windows

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#88] [Feature requests]
On MS Windows the app uses legacy title and scroll bar styles.  Wondering if it is possible to revamp. Generally, all apps in Windows now use skinny and themed scroll bars and a themed or non existent title bar. Mouse click and drag on the title bar zone still has the effect of being able to drag the window around.

**Comment by f1multiviewer** [comment]
Scroll bar styles have been fixed since, I think "Live stats" is best moved to it's own discussion, as it's completely unrelated.
